knowledge-base/basic-authentication-with-fiddler-everywhere.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Basic Authentication with Fiddler Everywhere
+description: This article briefs about the steps of basic authentication with fiddler everywhere.
+type: how-to
+slug: basic-authentication-with-fiddler-everywhere
+tags: basic authentication, fiddler everywhere, authentication with fiddler everywhere
+publish: true
+res_type: kb
+---
+
+## Environment
+
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>1.0.1</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Fiddler Everywhere</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+Fiddler Everywhere certificate inspects network traffic and debugs it. The __Composer__ feature in Fiddler Everywhere enables you to manually build and send HTTP, HTTPS requests. Click [here]({%slug composer%}) to know more about Fiddler Everywhere Composer.
+
+In the context of an HTTP transaction, __Basic Access Authentication__ is a method for an HTTP user agent (for example, a web browser) to provide a user name and password when making a request. In __Basic HTTP Authentication__, a request contains a header field in the form of Authorization: Basic <credentials>, where credentials are the __Base64__ encoding of `ID` and `password` joined by a single colon(`:`).
+
+Now, let's see how can you create a Composer request for APIs that require authentication:
+
+>tip In the steps shown below, you can use [httpbin.org](https://httpbin.org/) for HTTP request and response service.
+
+## Step 1
+
+Set the HTTP/HTTPS method to __GET__ and add the URL in the URL field. The URL added has been generated from the [httpbin.org](https://httpbin.org/).
+
+![Composer URL Field](../images/kb/url-field-of-composer-with-http-method.png)
+
+In the above image, as you can see that the URL specified is [https://httpbin.org/basic-auth/user1/pass1](https://httpbin.org/basic-auth/user1/pass1). Here __basic-auth__ determines that the Authentication taking place is Basic. The next two fields __user1__ and __pass1__ are the values of `user` and `passwd` variables, respectively.
+
+## Step 2
+
+Create an object inside the __Body__ tab. The predefined variables for the basic authentication are `user` and `passwd`. Set the values for both the variables.
```

Comment:
Set the values for both the variables. -> remove the second "the" -> Set the values for both variables.